### PR TITLE
feat!: upgrade ci, image, packages, audit

### PR DESCRIPTION
**Pull request by kleyow**
- Upgrade to node v16
- Update packages
- Update ci with dep/audit checks
- Standardize folder structure moving files out of /src
- Remove sub libraries for easier package maintenance
- Use `module-alias` for imports

Breaking change: Docker image folder structure no longer puts `/src` at the root and is now located in `opt/app/src/`. Depending on where you mount `rules.json` and where `RULES_FILE` env variable points to you will need to update one or the other.

Major version bump since this is a big upgrade.

### CODE REVIEW

**Comment by kleyow** [comment]
v13.0.0-snapshot.2 tested on `moja3`

```
NAME: moja3
LAST DEPLOYED: Sat Jul  9 21:34:59 2022
NAMESPACE: moja3
STATUS: deployed
REVISION: 1
TEST SUITE:     moja3-test
Last Started:   Sat Jul  9 21:48:45 2022
Last Completed: Sat Jul  9 21:48:57 2022
Phase:          Succeeded
TEST SUITE:     moja3-test
Last Started:   Sat Jul  9 21:48:57 2022
Last Completed: Sat Jul  9 21:49:08 2022
Phase:          Succeeded
TEST SUITE:     moja3-ml-ttk-test-setup
Last Started:   Sat Jul  9 21:44:09 2022
Last Completed: Sat Jul  9 21:45:00 2022
Phase:          Succeeded
TEST SUITE:     moja3-ml-ttk-test-val-bulk
Last Started:   Sat Jul  9 21:48:09 2022
Last Completed: Sat Jul  9 21:48:45 2022
Phase:          Succeeded
TEST SUITE:     moja3-ml-ttk-test-val-gp
Last Started:   Sat Jul  9 21:45:01 2022
Last Completed: Sat Jul  9 21:48:09 2022
Phase:          Succeeded
NOTES:
Thank you for installing mojaloop.
```

File: package.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
+{
+  "name": "mojaloop-simulator",
+  "version": "13.0.0-snapshot.2",
+  "description": "A canonical test example implementation of the parties, transfers and quotes resources of the Mojaloop API",
+  "license": "Apache-2.0",
+  "main": "src/index.js",
+  "author": "Matt Kingston, ModusBox Inc.",
+  "contributors": [
+    "Aaron Reynoza <aaron.reynoza@modusbox.com>",
+    "Kamuela Franco <kamuela.franco@modusbox.com>",
```
Comment (elnyry-sam-k) [inline package.json:10]:
```suggestion
    "Kamuela Franco <kamuela.franco@modusbox.com>",
     "Kevin Leyow <kevin.leyow@modusbox.com>",
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,100 @@
+{
+  "name": "mojaloop-simulator",
+  "version": "13.0.0-snapshot.2",
+  "description": "A canonical test example implementation of the parties, transfers and quotes resources of the Mojaloop API",
+  "license": "Apache-2.0",
+  "main": "src/index.js",
+  "author": "Matt Kingston, ModusBox Inc.",
```
Comment (elnyry-sam-k) [inline package.json]:
```suggestion
  "initial author": "Matt Kingston, ModusBox Inc.",
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,100 @@
+{
+  "name": "mojaloop-simulator",
+  "version": "13.0.0-snapshot.2",
+  "description": "A canonical test example implementation of the parties, transfers and quotes resources of the Mojaloop API",
```
Comment (elnyry-sam-k) [inline package.json]:
```suggestion
  "description": "A canonical test example implementation of the parties, transfers and quotes resources of the Mojaloop FSP Interoperability API",
```

**Review by elnyry-sam-k — DISMISSED** [review]

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**